### PR TITLE
fixed the css file for removing the hover effect from heading

### PR DESCRIPTION
**Pull request by TheAshutoshGupta**
Screenshot 2024-05-29 134356

Fixing #37

### CODE REVIEW

File: app/degreePlanner/detailPlanner.css [hunk 1]
```diff
@@ -147,9 +147,9 @@ i:hover {
     margin-left: 10px;
 
 }
-.semesterRow:hover {
+/* .semesterRow:hover {
```
Comment (jennifermtan) [inline app/degreePlanner/detailPlanner.css:150]: Hi, please delete the code instead of commenting it out.

**Review by jennifermtan — CHANGES_REQUESTED** [review]

**Comment by jennifermtan** [comment]
Hi @TheAshutoshGupta, please rename the PR to be more specific to the issue. Thanks :)

**Comment by TheAshutoshGupta** [comment]
> Hi @TheAshutoshGupta, please rename the PR to be more specific to the issue. Thanks :)

deleted the css lines, and also specified the PR name

**Review by jennifermtan — APPROVED** [review]
LGTM :)

**Comment by GoodGameRuler** [comment]
Approving changes. @TheAshutoshGupta please refer to branch naming rules. I will approve this PR however, next time I will require the branch naming scheme be followed.

**Pull request merged**